src/Chapter1.hs

```diff
@@ -508,7 +508,12 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = if absx > absy
+  then y
+  else x
+  where
+    absx = abs x
+    absy = abs y
```

Comment:
That's a nice solution using helper functions! But I guess this function can be a short one-liner :slightly_smiling_face: